docusaurus.config.js

```diff
@@ -2284,15 +2291,15 @@ const config = {
 						from: '/knowledgebase/troubleshooting',
 						to: '/en/faq/troubleshooting'
 					},
-          {
+					{
```

Comment:
It's IDE autoformatter. Should we add `prettify` to the doc? @justindeguzman

Reply:
I will add a linter at some point but need to do some code clean up first. A lot of the React files and JS config files are inconsistent.